templates/cicd/buildspec.yml

```diff
@@ -86,6 +88,14 @@ phases:
               build_args="$build_args--build-arg $arg "
             done
           fi
+          if [ -n "$build_target" ]; then
+            build_args="$build_args--target $build_target "
+          fi
+          if [ -n "$build_cache_from" ]; then
+            for arg in $(echo $build_cache_from | jq -r '.[]'); do
+              build_args="$build_args--cache-from $arg "
+            done
+          fi
```

Comment:
amazing! thank you so much for diving into this and figuring out how to parse an array